src/calculateRentalCost.js

```diff
@@ -4,7 +4,22 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  const priceADay = 40;
+  const LONG_TERM = 7;
+  const LONG_DISCOUNT = 50;
+  const MIDDLE_TERM = 3;
+  const MIDDLE_DISCOUNT = 20;
+  const TotalPrice = priceADay * days;
+
+  if (days >= MIDDLE_TERM && days < LONG_TERM) {
+    return TotalPrice - MIDDLE_DISCOUNT;
+  }
+
+  if (days >= LONG_TERM) {
+    return TotalPrice - LONG_DISCOUNT;
```

Comment:
The discount logic can be improved. Currently, if `days` is greater than or equal to `LONG_TERM`, only the `LONG_DISCOUNT` is applied. Consider restructuring the logic to ensure that the correct discount is applied based on the number of days.